src/commands/applications/list.ts

```diff
@@ -121,7 +135,9 @@ use applications:get to get full details for a single app.
 
   setRequestActionMessage(options: any) {
     debug('setRequestActionMessage()')
-    this.requestActionMessage = `Retrieving applications for organization ID ${chalk.cyan(options.organization)}`
+    options.organization ?
```

Comment:
Rewrite as to avoid repetitions

```
const optionalOrganizationInfo = options.organization ? ` for organization ID ${chalk.cyan(options.organization)}` : ''
this.requestActionMessage = 'Retrieving applications' + optionalOrganizationInfo
```